app/screens/authentication/Login.jsx

```diff
@@ -1,45 +1,86 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
-  Text, StyleSheet, View, Button, TouchableOpacity,
+  StyleSheet, View, Button,
 } from 'react-native';
+import { signInWithEmailAndPassword } from 'firebase/auth';
+import { LoginButton, LoginInput, LoginError } from './components';
+import { auth } from '../../firebaseConfig';
+import { COLORS } from '../../constants/colors';
 
-const styles = StyleSheet.create({
-  text: {
-    fontSize: 30,
-  },
-  touchable: {
-    width: 300,
-    marginTop: 556,
-    marginLeft: 25,
-    paddingVertical: 18,
-    borderWidth: 2,
-    borderColor: '#20232a',
-    borderRadius: 6,
-    backgroundColor: '#61dafb',
-    color: '#20232a',
-    textAlign: 'center',
-    fontSize: 30,
-    fontWeight: 'bold',
-  },
-});
+// console.log('Firebase Auth from login', auth);
+
+export default function Login({ navigation }) {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
```

Comment:
Don't know if i missed it but password state should be reset after a user successfully logs in. Generally, email and password should not be store in states but this is fine for now